application/microservices/entando-hub-catalog-ms/src/test/java/com/entando/hub/catalog/integration/CatalogFlowIT.java

```diff
@@ -194,7 +197,7 @@ void shouldReturnForbiddenWhenGettingWithNonAdminRole() throws Exception {
     void shouldReturnUnauthorizedWhenGettingWithNoRole() throws Exception {
```

Comment:
Please rename the method name with something like shouldReturnForbiddenWhenGettingWithNoRole